pkg/private/archive.py

```diff
@@ -11,53 +11,28 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Archive manipulation library for the Docker rules."""
+"""Archive reader library for the Docker rules."""
```

Comment:
Is this still docker-related?  Also, does this impact rules_docker?  IIRC they were (incorrectly) using this file.